libmusic/src/fft_wrapper.cpp

```diff
@@ -46,8 +46,10 @@ FFTWrapper::FFTWrapper(freq_hz_t f_low, freq_hz_t f_high, uint32_t sample_rate,
             FFTWrapper(f_low, f_high, BINS_PER_OCTAVE_DEFAULT, sample_rate,
                        win_size, hop_size) {}
 
-void FFTWrapper::Process(td_t td, uint32_t offset)
+void FFTWrapper::Process(const td_t & td, uint32_t offset)
 {
+    std::vector<fd_t> fft_sg;
```

Comment:
since `fft_sg` is used only for `LM_PEED` this should probably go under `#ifdef NDEBUG`

Reply:
this patch is better be dropped